src/BTVNanoCommissioning/workflows/ctag_eWc_valid_sf.py

```diff
@@ -539,6 +530,7 @@ def process_shift(self, events, shift_name):
                 out_branch, ["Jet_btagDeep*", "Jet_DeepJet*", "PFCands_*", "SV_*"]
             )
             # write to root files
+            print("Branches to write:", out_branch)
```

Comment:
Is this the check you want to keep when running the code? 
Indeed the `out_branch` also contains some regexp which not really returns full variable. Maybe would be interesting to check with [`fout["Events"].fields`](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/60/files#diff-48cce9500220df03b4e1eab1e318d64f22e159d7fdcd4df906bb9b5b7bdb85f7R546)  for check all the entries